frontend/packages/kubevirt-plugin/src/components/edit-button.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+
+export const EditButton: React.FC<EditButtonProps> = (props) => {
+  const { canEdit, onClick } = props;
+
+  if (canEdit) {
+    return (
+      <button
```

Comment:
Why not to use Button from PF4 ? I think console code is starting to use it too.

Reply:
done

Reply:
Moved back to `button` as styles got broken with moving the component to the "value section".
I would need to remove what PF4 Button is adding, so the "raw way" is simpler.